frontend/src/App.vue

```diff
@@ -1,28 +1,30 @@
 <template>
-  <div id="app">
-    <img alt="Vue logo" src="./assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js App"/>
-  </div>
+    <v-app id="app">
+        <top-navbar></top-navbar>
```

Comment:
it's probably better add LayoutPage with base scaffold there (including top navbar) and use use nested routing to register it